packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/bin/server/repositories/users_repository.dart

```diff
@@ -66,4 +70,24 @@ final Map<String, String> _unconfirmedPhoneNumbers = {};
 
   void deleteUser(String id, UserRole role) => _registeredUsers
       .removeWhere((user) => (user.id == id && user.role == role));
+
+  void setPasswordForUser(String email, String password) =>
+      _passwords[email] = password;
+
+  String? getPasswordForUser(String email) => _passwords[email];{{#enable_forgotten_password}}
+
+  bool isPasswordResetLockedForUser(String email) =>
+      _passwordResetLockedUsers.keys.contains(email);
+
+  int getPasswordResetTimeoutForUser(String email) =>
+      _passwordResetLockedUsers[email] ?? 60;
+
+  void lockPasswordResetForUser(String email) =>
+      _passwordResetLockedUsers[email] = 60;
```

Comment:
The `60` seconds value seems arbitrary. Can we introduce a constant so that we have a unique place where we can configure this, instead going manually over each occurrence?